### PR TITLE
CI: fix j2lint typo delimeter -> delimiter

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Fix typo in j2lint rule `jinja-statements-delimeter` vs `jinja-statements-delimiter` 

Update rule in templates that contain: `{# j2lint: disable=jinja-statements-delimiter #}`

Requires aristanetworks/j2lint#39 to be merged

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_designs/templates/interface_descriptions/underlay/ethernet-interfaces.j2
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_designs/templates/interface_descriptions/underlay/ethernet-interfaces.j2:2]: I believe this ({%-) is failing the CI.

**Comment by carlbuchmann** [comment]
Note CI: "Collection code testing / Validate j2 files (pull_request) " is currently failing which is normal until we merge aristanetworks/j2lint#39 

CI will need to be re-run once it is merged and should pass!

File: ansible_collections/arista/avd/roles/eos_designs/templates/interface_descriptions/underlay/ethernet-interfaces.j2
Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_designs/templates/interface_descriptions/underlay/ethernet-interfaces.j2:2]: Note CI: "Collection code testing / Validate j2 files (pull_request) " is currently failing which is normal until we merge https://github.com/aristanetworks/j2lint/pull/39

CI will need to be re-run once it is merged and should pass!

Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_designs/templates/interface_descriptions/underlay/ethernet-interfaces.j2:2]: I have added a commit in https://github.com/aristanetworks/j2lint/pull/39 that will allow to still support the legacy syntax hence decorrelating the two PRs. Once PR 39 has merged we can rerun the checks on this one but even if it merges and we don't merge this one immediately the rest of AVD CI should survive

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Review by gmuloc — COMMENTED** [review]
LGTM

**Review by gmuloc — APPROVED** [review]
LGTM (with approval..)

**Pull request merged**
